lib/src/components/NxTag/__tests__/NxTag.test.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import { shallow } from 'enzyme';
+import { faPlusCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+
+import NxTag, { NxSelectableTag } from '../NxTag';
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+describe('NxTag', function() {
+  const getShallowComponent = enzymeUtils.getShallowComponent(NxTag, { children: 'basic tag' });
+
+  it('renders NxTag with the `nx-tag` class along with any provided className', function() {
```

Comment:
This test isn't covering the "with any provided className" part so that should be removed from the name here.  It is being covered in a separate test below.

Reply:
> This test isn't covering the "with any provided className" part so that should be removed from the name here. It is being covered in a separate test below.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/3c1d663c9584b9f012d116df2a54843bb2b60e8a